Dockerfile.django

```diff
@@ -89,14 +89,12 @@ USER root
 RUN \
     adduser --system --no-create-home --disabled-password --gecos '' \
         --uid ${uid} ${appuser} && \
-    chown -R ${appuser} /app && \
-    chmod 0700 /app && \
-    chmod 0750 -R /app/* && \
-	chmod g=u /app && \
-	chmod -R g=u /app/* && \
+    chown -R root:root /app && \
```

Comment:
This should probably be kept as `appuser` so we don't get used to running everything as root

Reply:
@Maffooch not sure do I get your point. But intention to have it as a root is not to keep app files writable. While appuser can read/execute code